### ISSUE TITLE
4.1.18 errors

### ISSUE THREAD

**Issue by waggz81** [Chalsean/XFaction#328]
4.1.18 introduced an issue that caused our configuration to break. Errors start at line 65:

```
ID;Time;Cat;Vrb;Message;
1;11-22 23:06:44;;;Log for 'XFaction' created;
2;11-22 23:06:44;;;Format for log 'XFaction' is 'default';
3;11-22 23:06:44;Event;4;Started event listener [AddonEvent] for [ADDON_LOADED];
4;11-22 23:06:44;Event;3;Registered to receive ADDON_LOADED events;
5;11-22 23:06:44;Initialize;3;XFaction version [4.1.18];
6;11-22 23:06:44;RegionCollection;3;Initialized player region [1:US];
7;11-22 23:06:44;FactionCollection;3;Initialized faction [1:Alliance];
8;11-22 23:06:44;FactionCollection;3;Initialized faction [2:Horde];
9;11-22 23:06:44;FactionCollection;3;Initialized faction [3:Neutral];
10;11-22 23:06:44;RealmCollection;3;Initialized player realm [60:Stormrage];
11;11-22 23:06:44;Realm;4;-------------------------------------;
12;11-22 23:06:44;Realm;4;  key (string): Stormrage;
13;11-22 23:06:44;Realm;4;  name (string): Stormrage;
14;11-22 23:06:44;Realm;4;  initialized (boolean): true;
15;11-22 23:06:44;Realm;4;  ID (number): 60;
16;11-22 23:06:44;Realm;4;  apiName (nil): nil;
17;11-22 23:06:44;Realm;4;  connectedRealmCount (number): 0;
18;11-22 23:06:44;ZoneCollection;3;Initialized continent [Pandaria];
19;11-22 23:06:44;ZoneCollection;3;Initialized continent [Draenor];
20;11-22 23:06:44;ZoneCollection;3;Initialized continent [Broken Isles];
21;11-22 23:06:44;ZoneCollection;3;Initialized continent [Zandalar];
22;11-22 23:06:44;ZoneCollection;3;Initialized continent [Kul Tiras];
23;11-22 23:06:44;ZoneCollection;3;Initialized continent [Argus];
24;11-22 23:06:44;ZoneCollection;3;Initialized continent [Eastern Kingdoms];
25;11-22 23:06:44;ZoneCollection;3;Initialized continent [Kalimdor];
26;11-22 23:06:44;ZoneCollection;3;Initialized continent [Outland];
27;11-22 23:06:44;ZoneCollection;3;Initialized continent [Northrend];
28;11-22 23:06:44;ZoneCollection;3;Initialized continent [Nazjatar];
29;11-22 23:06:44;ZoneCollection;3;Initialized continent [Resonant Peaks];
30;11-22 23:06:44;ZoneCollection;3;Initialized continent [The Shadowlands];
31;11-22 23:06:44;ZoneCollection;3;Initialized zone [?];
32;11-22 23:06:44;Hook;4;Started hook [ReloadUI];
33;11-22 23:06:44;Hook;3;Hooked function ReloadUI;
34;11-22 23:06:44;Event;4;Started event listener [Logout] for [PLAYER_LOGOUT];
35;11-22 23:06:44;Event;3;Registered to receive PLAYER_LOGOUT events;
36;11-22 23:06:44;Event;4;Started event listener [LoadScreen] for [PLAYER_ENTERING_WORLD];
37;11-22 23:06:44;Event;3;Registered to receive PLAYER_ENTERING_WORLD events;
38;11-22 23:06:44;SystemEvent;3;Created CHAT_MSG_SYSTEM event filter;
39;11-22 23:06:44;Timer;4;Started timer [Login] for [1] seconds;
40;11-22 23:06:44;Event;4;Started event listener [DTTokenPLAYER_ENTERING_WORLD] for [PLAYER_ENTERING_WORLD];
41;11-22 23:06:44;Event;3;Registered to receive PLAYER_ENTERING_WORLD events;
42;11-22 23:06:44;DTToken;3;Registered for PLAYER_ENTERING_WORLD events;
43;11-22 23:06:44;Event;4;Started event listener [DTTokenPLAYER_LOGIN] for [PLAYER_LOGIN];
44;11-22 23:06:44;Event;3;Registered to receive PLAYER_LOGIN events;
45;11-22 23:06:44;DTToken;3;Registered for PLAYER_LOGIN events;
46;11-22 23:06:44;Event;4;Started event listener [DTTokenTOKEN_MARKET_PRICE_UPDATED] for [TOKEN_MARKET_PRICE_UPDATED];
47;11-22 23:06:44;Event;3;Registered to receive TOKEN_MARKET_PRICE_UPDATED events;
48;11-22 23:06:44;DTToken;3;Registered for TOKEN_MARKET_PRICE_UPDATED events;
49;11-22 23:06:44;Timer;4;Started timer [TokenTimer] for [60] seconds;
50;11-22 23:06:44;DTToken;3;New token price [206520];
51;11-22 23:06:44;ExpansionCollection;3;Initialized expansion [1:Retail];
52;11-22 23:06:44;ExpansionCollection;3;Initialized expansion [2:Classic];
53;11-22 23:06:44;Initialize;3;WoW client version [Retail:10.0.2];
54;11-22 23:06:44;AddonEvent;3;Addon is loaded and enabled [XFaction];
55;11-22 23:06:44;AddonEvent;3;Loading configs;
56;11-22 23:06:47;TimerEvent;4;Guild info is loaded, proceeding with setup;
57;11-22 23:06:47;Timer;4;Stopped timer [Login];
58;11-22 23:06:47;GuildCollection;3;Initialized guild [ZtH:Zeroes to Heroes];
59;11-22 23:06:47;GuildCollection;3;Initialized guild [DJ:Death Jesters];
60;11-22 23:06:47;GuildCollection;3;Initialized guild [LH:Little Heroes];
61;11-22 23:06:47;GuildCollection;3;Initialized guild [MH:Mini Heroes];
62;11-22 23:06:47;GuildCollection;3;Initialized guild [ZtHH:Zeroes to Hordies];
63;11-22 23:06:47;GuildCollection;3;Initialized guild [EDJ:Evil Death Jesters];
64;11-22 23:06:47;GuildCollection;3;Initialized guild [ZtHI:Zeroes to Heroes];
65;11-22 23:06:47;TimerEvent;1;Interface/AddOns/XFaction/Base/ObjectCollection.lua:79: attempt to index field 'objects' (a nil value);
66;11-22 23:06:47;TimerEvent;1;[string "@Interface/AddOns/XFaction/Developer/Logger.lua"]:29: in function `Error';
67;11-22 23:06:47;TimerEvent;1;[string "@Interface/AddOns/XFaction/Handlers/TimerEvent.lua"]:95: in function `catchBlock';
68;11-22 23:06:47;TimerEvent;1;[string "@Interface/AddOns/XFaction/Functions/Exception/try-catch-finally.lua"]:30: in function `catch';
69;11-22 23:06:47;TimerEvent;1;[string "@Interface/AddOns/XFaction/Handlers/TimerEvent.lua"]:94: in function <Interface/AddOns/XFaction/Handlers/TimerEvent.lua:21>;
70;11-22 23:06:47;TimerEvent;1;;
71;11-22 23:06:47;Event;4;Stopped event listener [AddonEvent] for [ADDON_LOADED];
72;11-22 23:06:47;Event;4;Stopped event listener [DTTokenTOKEN_MARKET_PRICE_UPDATED] for [TOKEN_MARKET_PRICE_UPDATED];
73;11-22 23:06:47;Event;4;Stopped event listener [DTTokenPLAYER_LOGIN] for [PLAYER_LOGIN];
74;11-22 23:06:47;Event;4;Stopped event listener [Logout] for [PLAYER_LOGOUT];
75;11-22 23:06:47;Event;4;Stopped event listener [LoadScreen] for [PLAYER_ENTERING_WORLD];
76;11-22 23:06:47;Event;4;Stopped event listener [DTTokenPLAYER_ENTERING_WORLD] for [PLAYER_ENTERING_WORLD];
77;11-22 23:06:47;Hook;4;Stopped hook [ReloadUI];
78;11-22 23:06:47;Timer;4;Stopped timer [Ping];
79;11-22 23:06:47;Timer;4;Stopped timer [Links];
80;11-22 23:06:47;Timer;4;Stopped timer [TokenTimer];
81;11-22 23:06:47;Timer;4;Stopped timer [Roster];
82;11-22 23:06:47;Timer;4;Stopped timer [DelayedLogin];
83;11-22 23:06:47;Timer;4;Stopped timer [Offline];
84;11-22 23:06:47;Timer;4;Stopped timer [Heartbeat];
85;11-22 23:06:47;Timer;4;Stopped timer [StaleLinks];
86;11-22 23:06:47;Timer;4;Stopped timer [Mailbox];
```

**Comment by Chalsean** [comment]
ill take a look once the servers are back up

**Comment by Chalsean** [comment]
this should be fixed in 4.1.20, give it a shot and let me know

**Comment by waggz81** [comment]
seems resolved
